docs/operation/operation.md

```diff
@@ -0,0 +1,236 @@
+    - [Stop score-server and clean up](#stop-score-server-and-clean-up)
+- [Integrating Keycloak](#integrating-keycloak)
+  - [Standalone](#standalone)
+  - [Docker](#docker)
+
+## Setting up the Development Environment
+
+### Standalone score-server
+
+#### Clone the Score Repository   
+
+Clone the Score repository to your local computer: [https://github.com/overture-stack/score.git](https://github.com/overture-stack/score.git)
+
+#### Build
+
+[JDK11](https://www.oracle.com/ca-en/java/technologies/downloads/) and [Maven3](https://maven.apache.org/download.cgi) are required to set up this service. To build the score-server run the following command from the score directory:
+
+```bash
+./mvnw clean install -DskipTests
+```
+
+#### Start the server
+
+Before running your score-server, ensure that your local machine is connected and running the following dependent services:
+
+- Song Server
+- Object Storage Service (S3 compatible storage or Azure)
+- Authentication/Authorization Service (EGO or Keycloak)
+
+Set the configuration of above dependent services on `score-server/src/main/resources/application.yml` and make sure to use the profiles according to your needs.
+
+**Profiles**
+| Profile | Description |
+| - | - |
+| *default* | Required to load default controllers |
+| *secure* | Required to load security configuration |
+| *collaboratory* or *azure* | Required to choose between S3 compatible or Azure storage |
+| *dev* | (Optional) to facilitate dev default configuration |
+
+Run the following command to start the score-server:
+
+```bash
+cd score-server/
+mvn spring-boot:run -Dspring-boot.run.profiles=default,collaboratory,secure,dev
+```
+
+>**Warning:** This guide is meant to demonstrate the configuration and usage of Score for development purposes and is not intended for production. If you ignore this warning and use this in any public or production environment, please remember to use Spring profiles accordingly. For production do not use ***dev*** profile.
+
+### Docker for Score
+
+Several `make` targets are provided for locally deploying the dependent services using Docker. As the developer, you can replicate a live environment for `score-server` and `score-client`. Using Docker allows you to develop locally, test submissions, create manifests, publish, unpublish and test score uploads/downloads in an isolated environment.
+
+For more information on the different targets, run `make help` or read the comments above each target for a description.
+
+> Note: We will need an internet connection for the `make` command, which may take several minutes to build. No external services are required for the `make` command.
+
+#### Start score-server and all dependencies.
+
+To start score-server and all dependencies, use the following command:
+
+```bash
+make clean start-score-server
+```
+
+#### Start the score-server (Mac M1 Users)
+
+On a Mac M1 you must set the Docker BuildKit environment variable to the legacy builder.
```

Comment:
Instead of M1 we should say Apple Silicon or ARM architecture, there are multiple generations of Mac's with this architecture now.